core/src/main/java/com/graphhopper/GraphHopper.java

```diff
@@ -977,8 +977,7 @@ public GHResponse route(GHRequest request) {
         readLock.lock();
         try {
             if (!encodingManager.supports(vehicle))
-                throw new IllegalArgumentException("Vehicle " + vehicle + " unsupported. "
-                        + "Supported are: " + getEncodingManager());
+                throw new IllegalArgumentException("Vehicle not supported: " + vehicle);
```

Comment:
I liked when router could inform easily in that message about the supported vehicles! :slightly_smiling_face:

Reply:
Done